avalanche.yaml

```diff
@@ -1,7 +1,38 @@
-componentCount: 50
-metricPrefix: "comp1"
-metricCount: 1000       # number of unique metrics
-labelCount: 100         # labels per metric
-valueInterval: 2      # values for metrics and labels refreshed every n seconds
+valueInterval: 2         # values for metrics and labels refreshed every n seconds
```

Comment:
Do you have an example of what the output would look like? Maybe add it as a git commit msg or something. I know at some point this will end up needing test.

Reply:
Added example entity in commit message.